robottelo/hosts.py

```diff
@@ -1288,3 +1288,21 @@ def update_setting(self, name, value):
         setting = self.api.Setting().search(query={'search': f'name="{name}"'})[0]
         setting.value = value
         setting.update({'value'})
+
+    def register_to_dogfood(self):
+        dogfood_canonical_hostname = settings.repos.dogfood_repo_host.partition('//')[2]
+        # get hostname of dogfood machine
+        dig_result = self.execute(f'dig +short {dogfood_canonical_hostname}')
+        # the host name finishes with a dot, so last character is removed
+        dogfood_hostname = dig_result.stdout.split()[0][:-1]
+        dogfood = Satellite(dogfood_hostname)
+        self.install_katello_ca(satellite=dogfood)
+        # satellite version consist from x.y.z, we need only x.y
+        sat_release = '.'.join(self.version.split('.')[:-1])
+        cmd_result = self.register_contenthost(
+            org='Sat6-CI', activation_key=f'satellite-{sat_release}-qa-rhel7'
```

Comment:
I don't think this org should be hardcoded. Ties into above comment on this method being a more generic registration method.

Reply:
Updated to use org from subscription.yaml